modules/start_text/settings.sqf

```diff
@@ -11,47 +11,53 @@
 //DATE -> Displays date (no text required)
 //DATETIME -> Displays date and time (no text required)
 
-
 switch (side player) do { //Checks what team the player is on
```

Comment:
an alternative to this would be to have separate setting arrays for the teams and doing the team switch inside the module init. Generally we don't want to expose anything but setting variable values to users.

Reply:
Fixed.